docs/developer-docs/gas-cost.md

```diff
@@ -177,22 +177,22 @@ The following table shows the calculated storage cost per GB for a 30-day month
 
 ## Monitoring canister cycle usage
```

Comment:
IMO these last 4 sections could be summarized under "Further readings" and then use a bullet list

## Further readings
* **Monitoring cycle usage**
   * [Check your canister's cycles balance](link)
   * [Monitor your canister's cycles usage](link)
   * [ Motoko cycles management library](link)
* **Topping up canisters**
   * ...

Reply:
Revised to use a bulleted list under 'Further readings'